doc/buckets.md

```diff
@@ -49,7 +50,7 @@ IMPORT_PROJECTID='myproject'
 
 * If `_PROJECTID` is not defined  env `GOOGLE_CLOUD_PROJECT` is used as default value
 
-* If neither _KEYFILENAME nor GOOGLE_APPLICATION_CREDENTIALS variable is provided (meaning the container is inheriting default credentials from an instance), the service account needs one additional permission at the project level: iam.serviceAccounts.signBlob (contained by the role roles/iam.serviceAccountTokenCreator)
+* If neither _KEYFILENAME nor GOOGLE_APPLICATION_CREDENTIALS variable is provided (meaning the container is inheriting default credentials from an instance), the service account needs one additional permission: iam.serviceAccounts.signBlob (contained by the role roles/iam.serviceAccountTokenCreator). Instructions above.
```

Comment:
I would say `See the instructions above`

and if you are going to change, `variable is provided ` -> `variables are provided`